crates/astria-core/src/sequencerblock/v1alpha1/block.rs

```diff
@@ -1318,17 +1320,17 @@ impl Deposit {
         amount: u128,
         asset: asset::Denom,
         destination_chain_address: String,
-        source_transaction_hash: String,
-        source_transaction_index: u32,
+        transaction_id: TransactionId,
+        index_of_action: u32,
```

Comment:
Maybe the following?
```suggestion
        position_in_source_action: u32,
```